.github/workflows/release.yml

```
@@ -37,70 +32,77 @@ jobs:
           git tag -a "${{ github.event.inputs.tag }}" -m "Release ${{ github.event.inputs.tag }}"
           git push origin "${{ github.event.inputs.tag }}"
 
-  latex:
-    runs-on: ubuntu-latest
+  build-latex:
+    uses: ./.github/workflows/latex.yml
```

actionlint (1.7.7)

error while parsing reusable workflow "./.github/workflows/latex.yml": "workflow_call" event trigger is not found in "on:" at line:4, column:3

(workflow-call)